src/sdk/object_io.js

```diff
@@ -211,6 +213,7 @@ class ObjectIO {
             'bucket',
             'key',
             'md_conditions',
+            'last_modified_time',
```

Comment:
Add also to create_params ... for symmetry.

Reply:
ok